src/main/resources/assets/purpurclient/lang/fr_fr.json

```diff
@@ -1,11 +1,23 @@
 {
   "purpurclient.options.title": "Paramètres de Purpur Client",
```

Comment:
Paramètres du client Purpur
Cela donne un résultat plus français.

Reply:
Purpur Client is a name mod